components/admin_console/admin_sidebar_header/admin_sidebar_header.tsx

```diff
@@ -3,8 +3,9 @@
 
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
+
 import {Client4} from 'mattermost-redux/client';
-import {UserProfile} from 'mattermost-redux/src/types/users';
+import {UserProfile} from 'mattermost-redux/types/users';
```

Comment:
A few of these files incorrectly imported things from mattermost-redux's src directory since importing from `mattermost-redux` takes it from `packages/mattermost-redux/src` directly now, that needs to be changed to prevent it from trying to look at `src/src`